string                  object_name
bool                    to_remove

---

bool is_success
